src/main/scala/com/cloudant/clouseau/IndexCleanupService.scala

```diff
@@ -62,4 +64,17 @@ class IndexCleanupService(ctx: ServiceContext[ConfigurationArgs]) extends Servic
       fileOrDir.delete
   }
 
+  private def move(path: String) {
+    val orgDir = new File(rootDir, path)
+    val targetPath = ".recovery" + File.separator + path
```

Comment:
we use `new File(parent, dir)` construction elsewhere when constructing nested paths, so let's do the same here for consistency.